src/pages/contributors.js

```diff
@@ -3,25 +3,109 @@ import Container from "../components/container"
 import Header from "../components/header"
 import Meta from "../components/metadata"
 import ExternalLink from "../components/externalLink"
+import GithubService from "../services/githubService"
 
-export default () => (
-  <Container>
-    <Meta pageTitle="Contributors" />
-    <Header />
-    <section className="hero is-danger">
-      <div className="hero-body">
-        <div className="container">
-          <h1 className="title">No contributors yet :(</h1>
-          <h2 className="subtitle">
-            Feel free to contribute by making Pull Request or suggesting things.
-          </h2>
-          <ExternalLink
+function Contributor(props) {
+  return (
+    <div className="card" key={props.html_url}>
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img src={props.avatar_url} alt="Placeholder image" />
+        </figure>
+      </div>
+      <div className="card-content">
+        <div className="media">
+          <div className="media-content">
+            <p className="title is-4">{props.name}</p>
+            <p className="subtitle is-6"><a href={props.url}>@{props.login}</a></p>
+          </div>
+        </div>
+
+        <div className="content">
+          {props.bio}
```

Comment:
Why bio isn't around `<p>` and contributions yes, I don't think contributions should be around `p`